README.md

```diff
@@ -196,6 +196,42 @@ To make this process secure, Guardian should only be run in `internal` or
 but still ensures only the approved and merged workflow is being executed for
 the pull request.
 
+## Workload Identity Federation
+Terraform service accounts have elevated privileges, following [WIF Best Practices](https://cloud.google.com/iam/docs/best-practices-for-using-workload-identity-federation)
+is recommended when setting up guardian to use Workload Identity Federation. Below is an example attribute
+configuration that can be used to set up guardian in a secure manner.
+
+The following [attribute mappings](https://cloud.google.com/iam/docs/workload-identity-federation#mapping) map claims from the [GitHub Actions JWT](https://token.actions.githubusercontent.com/.well-known/openid-configuration)
+to Google STS token attributes.
+  * `google.subject=assertion.sub`
+  * `attribute.actor=assertion.actor`
+  * `attribute.aud=assertion.aud`
+  * `attribute.repository_owner_id=assertion.repository_owner_id`
+  * `attribute.repository_id=assertion.repository_owner_id`
+  * `attribute.repository_visibility=assertion.repository_visibility`
+  * `attribute.workflow_ref=assertion.workflow_ref`
```

Comment:
Added ref to support the expression below:

```shell
  * `google.subject=assertion.sub`
  * `attribute.actor=assertion.actor`
  * `attribute.aud=assertion.aud`
  * `attribute.ref=assertion.ref`
  * `attribute.repository_owner_id=assertion.repository_owner_id`
  * `attribute.repository_id=assertion.repository_owner_id`
  * `attribute.repository_visibility=assertion.repository_visibility`
  * `attribute.workflow_ref=assertion.workflow_ref`
```

Reply:
added `attribute.ref=assertion.ref` to the list